.github/workflows/nodejs.yml

```diff
@@ -0,0 +1,25 @@
+# This workflow will do a clean install of node dependencies, build the source code and run tests across different versions of node
+name: Node.js CI
+
+on:
+  [push, pull_request]
+
+jobs:
+  build:
+
+    runs-on: [ubuntu-latest, macos-latest, ubuntu-16.04]
+
+    strategy:
+      matrix:
+        node-version: [12.x, latest]
+
+    steps:
+    - uses: actions/checkout@v2
+    - name: Use Node.js ${{ matrix.node-version }}
+      uses: actions/setup-node@v1
+      with:
+        node-version: ${{ matrix.node-version }}
+    - run: npm run lint
```

Comment:
It's better to separate lint into a single action - this way it will run only once on a selected version of Node/OS and give faster and more prominent feedback to a contributor.

Reply:
I agree. Could we make it a separate workflow? First node then lint?

Reply:
Yea, Should I make it two files?

Reply:
Yes, that what I mean

Reply:
@NotMoni Yes make it into 2 files.

Reply:
Sure 👍